delira/data_loading/data_manager.py

```diff
@@ -245,108 +244,111 @@ def n_batches(self):
             raise ValueError('Invalid value for n_process_augmentation')
         return n_batches
 
+import os
+if "torch" in os.environ["DELIRA_BACKEND"]:
 
-class ConcatDataManager(object):
-    """
-    Class to concatenate DataManagers
-
-    """
-
-    def __init__(self, datamanager=typing.List[BaseDataManager]):
-        """
-
-        Parameters
-        ----------
-        datamanager : list
-            the datamanagers which should be concatenated
-            (All attributes except the dataset are extracted 
-            from the first manager inside the list)
-
-        """
-
-        self.dataset = ConcatDataset(
-            [tmp.dataset for tmp in datamanager])
-
-        self.data_loader_cls = datamanager[0].data_loader_cls
-
-        self.batch_size = datamanager[0].batch_size
-        self.n_process_augmentation = datamanager[0].n_process_augmentation
-        self.transforms = datamanager[0].transforms
-        self.sampler = datamanager[0].sampler.__class__.from_dataset(
-            self.dataset
-        )
-
-    def get_batchgen(self, seed=1):
+    from torch.utils.data import ConcatDataset
+    class ConcatDataManager(object):
```

Comment:
in what way does ConcatDataManager depend on `pytorch`?

Reply:
Looking at the pytorch docs, [ConcatDataset](https://pytorch.org/docs/stable/_modules/torch/utils/data/dataset.html#ConcatDataset) itself also doesn't depend on any torch-specific functions. Imo we should add a custom implementation.

Reply:
I'll do that today or tomorrow

Reply:
Should be fixed by 8e364e5b374a0656bce307fe445931b6a3057e8c